warehouse/models/payments_views/payments_tests_date_spine.sql

```diff
@@ -0,0 +1,31 @@
+{{ config(materialized='ephemeral') }}
+
+WITH payments_rides AS (
+    SELECT * FROM {{ ref('payments_rides') }}
+),
+
+distinct_providers AS (
+    SELECT DISTINCT participant_id
+    FROM payments_rides
+),
+
+min_max_dates AS (
+    SELECT
+        MIN(DATE(transaction_date_time_pacific)) AS min_date,
+        MAX(DATE(transaction_date_time_pacific)) AS max_date
+    FROM payments_rides
+),
+
+create_date_range AS (
+        {{ dbt_date.get_base_dates(start_date="2022-01-01", end_date="2022-02-01", datepart="day") }}
```

Comment:
@atvaccaro okay I hardcoded these dates into the macro here to make sure it works as expected but I'm having a hard time using the `max`/`min` values from the previous CTE for `start_date` and `end_date` - it seems to want to explicit dates? this is probably a dumb question but whats the best way to use the calculated values here?

Reply:
Ah yeah you'd have to get the values as strings in Jinja using `get_column_values` and template them in. I think it's fine to leave as you had it, unless you want to practice using macros.

Reply:
Actually on second though, I think you may be able to just select them as long as the value is properly parenthetical? see https://github.com/dbt-labs/dbt-utils#date_spine-source